### PR TITLE
add a files param to TestRunner.test

### PR DESCRIPTION
**Pull request by devoncarew**
- add a `files` param to `TestRunner.test`
- publish a patch version

### CODE REVIEW

File: lib/grinder_tools.dart [hunk 1]
```diff
@@ -217,9 +225,11 @@ class TestRunner {
   /// `firefox`, `safari`. [concurrency] controls the number of concurrent test
   /// suites run (defaults to 4). [pubServe] is the port of a pub serve instance
   /// serving `test/`.
-  Future testAsync({String name, String plainName, dynamic platformSelector,
-      int concurrency, int pubServe, RunOptions runOptions}) {
+  Future testAsync({dynamic files, String name, String plainName,
+      dynamic platformSelector, int concurrency, int pubServe,
+      RunOptions runOptions}) {
     return _test.runAsync(_buildArgs(
+        files: files,
```
Comment (seaneagan) [inline lib/grinder_tools.dart]: An optional positional parameter would be more consistent with the named command-line test runner and with `Dartfmt` and `Analyzer`, but we'd have to wait until [this DEP](https://github.com/lrhn/dep-optionals) until we could actually make it optional.  If we keep it named, maybe `paths` is more self-documenting that you can use a directory path and not just file paths?

Comment (devoncarew) [inline lib/grinder_tools.dart]: Totally agree re: the optional positional parameter; would have preferred to have something like that available. I think the primary use-case will be files however; would prefer to keep it as is.

Comment (seaneagan) [inline lib/grinder_tools.dart]: sgtm

File: lib/grinder_sdk.dart [hunk 2]
```diff
@@ -305,7 +306,7 @@ class Analyzer {
     List args = [];
     if (packageRoot != null) args.add('--package-root=${packageRoot.path}');
     if (fatalWarnings) args.add('--fatal-warnings');
-    args.addAll(_coerceToPathList(fileOrPaths));
+    args.addAll(coerceToPathList(fileOrPaths));
```
Comment (zoechi) [inline lib/grinder_sdk.dart]: It's a bit weird that here are `fileOrPaths`, `files` (line 320), `fileOrPath` (line 335) for the same kind of argument. Also the doc-comments are written differently.

**Pull request merged**